test/buy-tshirt.spec.ts

```diff
@@ -22,8 +25,14 @@ describe('Buy a t-shirt', () => {
 
     await menuContentPage.goToTShirtMenu();
 
+    await browser.wait(
```

Comment:
I think we can encapsulate the explicit wait logic on each page object. This will leave the test class without the responsibility to wait for elements that the tests should not take care of. Indeed, this is one of the advantages of POM.